ang/crmRouteBinder.js

```diff
@@ -50,18 +50,18 @@
         registerGlobalListener($injector);
 
         options.format = options.format || 'json';
-        var fmt = formats[options.format];
+        var fmt = _.clone(formats[options.format]);
         if (options.deep) {
           fmt.watcher = '$watch';
```

Comment:
Adding `_.clone()` on l53 prevents line 55 here from modifying the global defaults.